# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.cpp
# Commit: 6a43f9853075
# Full Hash: 6a43f985307516ec1ae2d413a39cd6d813560b8b
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2020-05-06 09:37:16
# Regressor Bug: 1574746
# File Overlap Count: 1
# Description:
#   Bug 1574746 - Remove AllowWebRenderForThisWindow() r=nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71440
# ==============================================================================

diff -r 74d50028caec -r 6a43f9853075 widget/gtk/nsWindow.cpp
--- a/widget/gtk/nsWindow.cpp	Tue May 05 23:57:55 2020 +0000
+++ b/widget/gtk/nsWindow.cpp	Tue May 05 23:44:36 2020 +0000
@@ -4161,11 +4161,8 @@
       // gfxVars, used below.
       Unused << gfxPlatform::GetPlatform();
 
-      bool useWebRender =
-          gfx::gfxVars::UseWebRender() && AllowWebRenderForThisWindow();
-
       mIsAccelerated = ComputeShouldAccelerate();
-      MOZ_ASSERT(mIsAccelerated | !useWebRender);
+      bool useWebRender = gfx::gfxVars::UseWebRender() && mIsAccelerated;
 
       if (mWindowType == eWindowType_toplevel) {
         // We enable titlebar rendering for toplevel windows only.